wgengine/magicsock/magicsock.go

```diff
@@ -67,6 +67,7 @@ type Conn struct {
 	//	10.0.0.3:3 -> [10.0.0.3:3]
 	addrsMu    sync.Mutex
 	addrsByUDP map[udpAddr]*AddrSet
+	addrsByKey map[key.Public]*AddrSet
```

Comment:
add same-line `// TODO: clean this map up sometime?` comments to both of these map fields?